background.js

```diff
@@ -209,6 +209,9 @@ $(document).ready(function () {
           // chrome.tabs.create({ 'url': 'https://www.youtube.com/results?search_query=' + data.result.parameters.any });
                 } else if (data.result.metadata.intentName === "open") {
                    chrome.tabs.create({ 'url': "http://www." + data.result.parameters.website });
+                } else if (data.result.metadata.intentName === "incognito"||data.result.metadata.intentName === "safemode"||data.result.metadata.intentName === "privatebrowsing"||data.result.metadata.intentName === "openinsafemode") {
+                   chrome.windows.create({url:"http://www.google.com",incognito: true});
+                   Speech("Have   you   allowed  me  to   open   in   incognito  mode ?    If not   go  to chrome extension and give the permission");                   
```

Comment:
Should this be only said when it doesnt open in incognito. can you make like that .everything else looks good

Reply:
speech () was added to inform the user he /she must allow the permision to anna extension to allow in incognito..
So I thought it is a must line..
Should I remove it ?

Reply:
I think that speech should only come ,if they dont allow incognito